osu.Game/Database/ArchiveModelManager.cs

```diff
@@ -44,7 +44,7 @@ public abstract class ArchiveModelManager<TModel, TFileModel> : ICanAcceptFiles
         /// Fired when a new <see cref="TModel"/> becomes available in the database.
         /// This is not guaranteed to run on the update thread.
         /// </summary>
-        public event ItemAddedDelegate ItemAdded;
+        public event Action<TModel, bool> ItemAdded;
```

Comment:
intentional change? if so, `ItemAddedDelegate` should be removed.

Reply:
still not sure about this one. the reason it had a delegate is that the bool here is ambiguous.

Reply:
there wasn't really any particular reason for changing this other than avoiding declaring a delegate in `IModelManager`. should it be added back on the interface or should we just add it to the documentation?